browser/src/control/Control.SheetsBar.js

```diff
@@ -34,6 +34,8 @@ class SheetsBar {
 		this.create();
```

Comment:
Above you see `this.builder` creation, you need custom callback to implement onSelected action for your menu.
Like in: https://github.com/CollaboraOnline/online/blob/23e4ca68c177338b0cfa6512e39bc2b63f80f11c/browser/src/control/jsdialog/Component.Toolbar.ts#L49

Implementation looks like:
https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.FormulaBarJSDialog.js#L179

You need to react when `id` matches your widget and you can read data of the selection event.
If it is a call for not your submenu, you just call default callback